wazo_calld/plugin_helpers/ari_.py

```diff
@@ -155,6 +155,14 @@ def __init__(self, channel_id, ari):
     def __str__(self):
         return self.id
 
+    def asterisk_name(self):
```

Comment:
Out of curiosity, what is the difference between `asterisk_name` vs `name`?
Is it because `name` can be confused with channel id?